src/plots/cartesian/index.js

```diff
@@ -560,20 +603,6 @@ function makeSubplotLayer(gd, plotinfo) {
 
     // common attributes for all subplots, overlays or not
 
-    if(!hasOnlyLargeSploms) {
```

Comment:
Thanks for the PR.
Could you please why `splom_large` mock is not rendered now?

Reply:
splom_large did render, however the clear afterwards had a problem. I fixed it now.